### PR TITLE
Fix docs for build-cross-osx and build-cross-arm64

### PR DESCRIPTION
**Pull request by abyss7**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation

### CODE REVIEW

File: docs/en/development/build_cross_arm.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# How to Build ClickHouse on Linux for AARCH64 (ARM64) architecture
+
+This is for the case when you have Linux machine and want to use it to build `clickhouse` binary that will run on another Linux machine with AARCH64 CPU architecture. This is intended for continuous integration checks that run on Linux servers.
+
```
Comment (BayoNet) [inline docs/en/development/build_cross_arm.md]: I didn't understand relation between the ClickHouse building and CI that run on some unknown Linux servers.

File: docs/en/development/build_cross_arm.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+The cross-build for AARCH64 is based on the Build instructions, follow them first.
+
```
Comment (BayoNet) [inline docs/en/development/build_cross_arm.md]: Void text.

File: docs/en/development/build_cross_arm.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+# Install Clang-8
+
+Follow the instructions from https://apt.llvm.org/ for your Ubuntu or Debian setup.
+For example the commands for Bionic are like:
```
Comment (BayoNet) [inline docs/en/development/build_cross_arm.md]:
```suggestion
For example, in Ubuntu Bionic you can use the following commands:
```

File: docs/en/development/build_cross_osx.md [hunk 4]
```diff
@@ -31,22 +31,26 @@ git clone https://github.com/tpoechtrager/cctools-port.git
 cd cctools-port/cctools
 ./configure --prefix=${CCTOOLS} --with-libtapi=${CCTOOLS} --target=x86_64-apple-darwin
 make install
+```
+
+Also we need to download MacOS X SDK inside our working tree
```
Comment (BayoNet) [inline docs/en/development/build_cross_osx.md]:
```suggestion
Also, we need to download MacOS X SDK inside our working tree.
```

File: docs/en/development/build_cross_arm.md (hunk 2 above)
Comment (abyss7) [reply docs/en/development/build_cross_arm.md]: What is an expected action point?

File: docs/en/development/build_cross_arm.md (hunk 1 above)
Comment (abyss7) [reply docs/en/development/build_cross_arm.md]: The point is that the purpose of this instruction is for some CI Build to work on Linux server (with x86-64 architecture). If you can rephrase it in a better way, you're welcome.

**Pull request merged**